resource-aggregate/pb/commands.proto

```diff
@@ -385,3 +385,96 @@ message ConfirmResourceCreateResponse {
     AuditContext audit_context = 100;
 }
 
+// Update Device Metadata - Connection status                           
```

Comment:
Please create [here](https://github.com/plgd-dev/plgd.dev/tree/main/content/en/features) `device-shadow.md` and document what is it, how it works, how user can control it. (document feature itself including recent changes).